config.development.json

```diff
@@ -0,0 +1,6 @@
+{
+  "enableDeveloperExperiments": true,
+  "server": {
+    "host": "0.0.0.0"
+  }
+}
```

Comment:
This should not be in the commit

Reply:
oh sorry my bad
it wasn't there before though
I'll remove it